save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ProjectController.kt

```diff
@@ -197,6 +192,27 @@ class ProjectController(
             ResponseEntity.ok(projectStatus.message)
         }
 
+    /**
+     * @param project
+     * @param authentication
+     * @return response
+     */
+    @PostMapping("/delete")
+    fun deleteProject(@RequestBody project: Project, authentication: Authentication): Mono<StringResponse> =
+            projectService.findWithPermissionByNameAndOrganization(
+                authentication, project.name, project.organization, Permission.DELETE
+            )
+                .filter { projectPermissionEvaluator.hasPermission(authentication, project, Permission.DELETE) }
```

Comment:
Isn't it duplication of `findWithPermission...`?

Reply:
yep, seems like it is